amd/src/annoto.js

```diff
@@ -168,6 +168,7 @@ define([
         bootWidget: function() {
             var params = this.params;
             var config = {
+                deploymentDomain: params.deploymentDomain,
```

Comment:
@ilyaDymshits Please update the configuration to be like :
config.backend = {
                    domain: params.deploymentDomain,
                };
https://github.com/Annoto/widget-api/blob/ea2f11b5aa13abbb46d55b2e664a565e6dde842e/lib/config.d.ts#L380